plugins/sigma.layout.fruchtermanReingold/sigma.layout.fruchtermanReingold.js

```diff
@@ -0,0 +1,285 @@
+;(function(undefined) {
+  'use strict';
+
+  if (typeof sigma === 'undefined')
+    throw new Error('sigma is not declared');
+
+  // Initialize package:
+  sigma.utils.pkg('sigma.layouts.fruchtermanReingold');
+
+  /**
+   * Sigma Fruchterman-Reingold
+   * ===============================
+   *
+   * Author: Sébastien Heymann @ Linkurious
+   * Version: 0.1
+   */
+
+
+  var settings = {
+    autoArea: true,
+    area: 1,
+    gravity: 10,
+    speed: 0.1,
+    maxIterations: 1000
+  };
+
+  /**
+   * Event emitter Object
+   * ------------------
+   */
+  var eventEmitter = {};
+  sigma.classes.dispatcher.extend(eventEmitter);
+
+  /**
+   * Fruchterman Object
+   * ------------------
+   */
+  function FruchtermanReingold() {
+  };
+
+  FruchtermanReingold.prototype.init = function (sigInst, options) {
```

Comment:
Using a method linked to the prototype of the object can be tricky here. A sigma instance is passed as an argument and stored within the prototype. As a consequence all instances of the layout will have the same sigma instance reference. Just like a `static` variable in Java.

Is that on purpose ?
(This can't be used on several instances at the same time I think)